src/transformers/modeling_bert.py

```diff
@@ -922,17 +922,22 @@ def forward(
         position_ids=None,
         head_mask=None,
         inputs_embeds=None,
-        labels=None,
         output_attentions=None,
         output_hidden_states=None,
         adapter_names=None,
         head=None,
         return_dict=None,
+        **kwargs
     ):
         input_ids = input_ids.view(-1, input_ids.size(-1)) if input_ids is not None else None
         attention_mask = attention_mask.view(-1, attention_mask.size(-1)) if attention_mask is not None else None
         token_type_ids = token_type_ids.view(-1, token_type_ids.size(-1)) if token_type_ids is not None else None
         position_ids = position_ids.view(-1, position_ids.size(-1)) if position_ids is not None else None
+        inputs_embeds = (
+            inputs_embeds.view(-1, inputs_embeds.size(-2), inputs_embeds.size(-1))
+            if inputs_embeds is not None
+            else None
+        )
```

Comment:
Why is this reformatting now necessary?

Reply:
I believe it was already necessary before but the change got lost during syncing with HF changes (I think the parameter was introduced in HF 3.x).

Reply:
* also, the code is just copied from `BertForMultipleChoice`

Reply:
That makes sense